src/zoid/auth/component.jsx

```diff
@@ -62,93 +72,119 @@ export function getAuthComponent() : ZoidComponent<AuthPropsType> {
                     required:   false
                 },
 
-                clientID: {
-                    type:       'string',
-                    value:      () => getClientID()
+                client_id: {
+                    type:               'string',
+                    queryParam:         true,
+                    value:      () =>   getClientID(),
+                    required:           false
                 },
-        
+
                 sessionID: {
                     type:       'string',
                     value:      getSessionID,
                     queryParam: false
                 },
-        
+
                 authSessionID: {
                     type:       'string',
                     queryParam: false,
                     required:   false
                 },
-                
+                scope: {
+                    type:       'string',
+                    queryParam: true,
+                    required:   false
+                },
+                redirect_uri: {
+                    type:       'string',
+                    queryParam: true,
+                    required:   false
+                },
                 env: {
                     type:       'string',
                     queryParam: true,
                     value:      getEnv
                 },
-        
                 sdkMeta: {
                     type:       'string',
                     queryParam: true,
                     value:      getSDKMeta
                 },
-        
+
                 nonce: {
-                    type:     'string',
-                    required: false,
-                    value:    getCSPNonce
+                    type:               'string',
+                    queryParam:         true,
+                    value:      () =>   getCSPNonce()
                 },
 
-                buyerCountry: {
-                    type:       'string',
-                    queryParam: true,
-                    required:   false,
-                    default:    getBuyerCountry
-                },
-        
-                locale: {
-                    type:          'object',
-                    queryParam:    'locale.x',
-                    allowDelegate: true,
-                    queryValue:    ({ value }) => `${ value.lang }_${ value.country }`,
-                    value:         getLocale
-                },
 
-                version: {
-                    type:       'string',
-                    queryParam: true,
-                    value:      getVersion
-                },
+                // responseType: {
```

Comment:
Why is this commented out? responseType is required right?

Reply:
i have reverted it bak, it was part of my testing but accidently it checked in